src/video/video.js

```diff
@@ -31,32 +33,36 @@ export default (
     BaseComponent,
     mapStateToProps = defaultMapStateToProps,
     mapVideoElToProps = defaultMapVideoElToProps,
-    mergeProps = defaultMergeProps
+    mergeProps = defaultMergeProps,
+    videoId
```

Comment:
we have `videoId` here and `mediaId` elsewhere -- I think we should standardize around `videoId`